websites/D/dアニメストア/dist/metadata.json

```diff
@@ -12,8 +12,8 @@
 		"pt_BR": "Site número 1 em streaming de animes no Japão! Grátis no primeiro mês! Assista mais de 2,900 animes por 400 ienes por mês (excluindo taxas)! Você pode usar outros dispositivos móveis além de DoCoMo. Melhor qualidade em tablets, PCs, Chromecasts, e Fire TV!",
 		"nl": "De beste anime streaming website in Japan! Eerste maand gratis! Onbeperkt bekijken van meer dan 2900 animes voor 400 yen per maand (exclusief belasting)! U kunt andere dan DoCoMo mobiele apparaten ook gebruiken. Nog betere kwaliteit op tablets, pc's, Chromecasts en Fire TV!"
 	},
-	"url": "anime.dmkt-sp.jp",
-	"version": "1.2.11",
+	"url": "animestore.docomo.ne,jp",
+	"version": "2.1.0",
```

Comment:
I wouldn't quite consider this a breaking change, but a bit of a smaller update, so a version like `1.2.12` or `1.3.0` might make more sense.